.drone.yml

```diff
@@ -325,3 +325,182 @@ volumes:
   - name: cache
     host:
       path: /tmp/drone-cache
+---
+kind: pipeline
+type: docker
+name: php-continous-package
+
+trigger:
+  repo:
+    - friendica/friendica
+  branch:
+    - develop
+    - 20*-rc
+  event:
+    - push
+
+node:
+  node: releaser # This prevents executing this pipeline at other servers than drone.friendi.ca
+
+steps:
+  - name: Clone addon
+    image: alpine/git
+    commands:
+      - git clone https://github.com/friendica/friendica-addons ./addon
+      - cd ./addon
+      - git checkout $DRONE_REPO_BRANCH
+  - name: Restore cache
+    image: meltwater/drone-cache:dev
+    settings:
+      backend: "filesystem"
+      restore: true
+      cache_key: '{{ .Repo.Name }}_php74_{{ arch }}_{{ os }}'
+      archive_format: "gzip"
+      mount:
+        - '.composer'
+    volumes:
+      - name: cache
+        path:
+          /tmp/cache
+  - name: Composer install
+    image: friendicaci/php7.4:php7.4.18
+    commands:
+      - export COMPOSER_HOME=.composer
+      - composer validate
+      - composer install --no-dev --optimize-autoloader
+  - name: Create artifacts
+    image: debian
+    commands:
+      - apt-get update
+      - apt-get install bzip2
+      - export VERSION="$(cat VERSION)"
+      - export RELEASECORE="friendica-full-$VERSION"
+      - export ARTIFACT="$RELEASECORE.tar.gz"
+      - # Create artifact
+      - mkdir ./build
+      - tar -cvjf ./build/$ARTIFACT --transform "s,^,$RELEASECORE/," -X mods/release-list-exclude.txt -T mods/release-list-include.txt
+      - # calculate SHA256 checksum
+      - sha256sum ./build/$ARTIFACT > ./build/$ARTIFACT.sha256
```

Comment:
For me the additional `build` directory in the patch is somehow disturbing.

```
$> cat friendica-full-2021.09-rc.tar.gz.sha256; sha256sum friendica-full-2021.09-rc.tar.gz
df77fc21c6ccc224e6f548a137e0289a87dacd945fb0d3600f049570fdcc22bf  ./build/friendica-full-2021.09-rc.tar.gz
df77fc21c6ccc224e6f548a137e0289a87dacd945fb0d3600f049570fdcc22bf  friendica-full-2021.09-rc.tar.gz
```

and the same for the addon archive. Maybe the sum should be generated in the build directory.